packages/lesswrong/server/scripts/mergeAccounts.ts

```diff
@@ -420,12 +448,15 @@ Vulcan.mergeAccounts = async ({sourceUserId, targetUserId, dryRun}:{
 async function recomputeKarma(userId: string) {
   const user = await Users.findOne({_id: userId})
   if (!user) throw Error("Can't find user")
-  const allTargetVotes = await Votes.find({
+  const selector: Record<string, any> = {
     authorIds: user._id,
     userId: {$ne: user._id},
-    legacy: {$ne: true},
     cancelled: false
-  }).fetch()
+  };
+  if (!Votes.isPostgres()) {
+    selector.legacy = {$ne: true};
+  }
```

Comment:
Why does `Votes.isPostgres` affect this?

Reply:
`legacy` is an old field that no longer exists in the schema and, by extension, not in Postgres. This still isn't doing the right thing though - I've just changed it so we still check the value by reading it from `legacyData` in Postgres. Good catch!